go/dpagg/sum_test.go

```diff
@@ -1050,19 +1050,22 @@ func TestSumComputeConfidenceIntervalForInt64Computation(t *testing.T) {
 // Tests that ComputeConfidenceInterval for float64 returns a correct interval for a given sum.
 func TestSumComputeConfidenceIntervalForFloat64Computation(t *testing.T) {
 	for _, tc := range []struct {
-		desc string
-		opt  *BoundedSumFloat64Options
-		want noise.ConfidenceInterval
+		desc      string
+		opt       *BoundedSumFloat64Options
+		want      noise.ConfidenceInterval
+		tolerance float64
 	}{
 		{
-			desc: "Gaussian",
-			opt:  &BoundedSumFloat64Options{Epsilon: 0.1, Delta: 0.1, Lower: 0, Upper: 5, Noise: getNoiselessConfInt(noise.Gaussian())},
-			want: noise.ConfidenceInterval{LowerBound: 15.8964252365, UpperBound: 35.1035747635},
+			desc:      "Gaussian",
+			opt:       &BoundedSumFloat64Options{Epsilon: 0.1, Delta: 0.1, Lower: 0, Upper: 5, Noise: getNoiselessConfInt(noise.Gaussian())},
+			want:      noise.ConfidenceInterval{LowerBound: 15.898893, UpperBound: 35.101107},
+			tolerance: 1e-3,
 		},
 		{
-			desc: "Laplace",
-			opt:  &BoundedSumFloat64Options{Epsilon: 0.1, Lower: 0, Upper: 5, Noise: getNoiselessConfInt(noise.Laplace())},
-			want: noise.ConfidenceInterval{LowerBound: 0.0000000000, UpperBound: 60.1573590280},
+			desc:      "Laplace",
+			opt:       &BoundedSumFloat64Options{Epsilon: 0.1, Lower: 0, Upper: 5, Noise: getNoiselessConfInt(noise.Laplace())},
```

Comment:
Noise: noNoise{noise.Laplace()} maybe?

getNoislessConfInt is not a great name, you return not a confidence interval, but noise (so either not use extra method or rename it)

Reply:
Yeah makes sense, This function was added a month ago though. The reason to have a function in the first place is if you don't initialize noNoise struct with an actual noise object, you will get a Go's version of "nullpointerexception" so I made this function as a sort of constructor to force that you need to initialize it with either laplace or gaussian.
https://github.com/google/differential-privacy/blob/2ccce87b6f4508efd991732f7227dcda3cd39fb2/go/dpagg/dpagg_test.go#L58-L70

I am not sure of another name that conveys this "get a Noiseless interface for confidence intervals using X distribution".

Reply:
Removed the function and replaced it with noNoise{noise} in each reference.